sdk/textanalytics/ai-text-analytics/review/ai-text-analytics.api.md

```diff
@@ -247,6 +256,42 @@ export interface ExtractKeyPhrasesSuccessResult extends TextAnalyticsSuccessResu
     keyPhrases: string[];
 }
 
+// @public
+export interface ExtractSummaryAction extends TextAnalyticsAction {
```

Comment:
ExtractSummarySentencesAction

Reply:
I prefer the brevity of `ExtractSummaryAction`, but could be convinced otherwise. Also it seems if we name this `ExtractSummarySentencesAction` then the result object should also be renamed `ExtractSummarySentencesActionResult` to be consistent.

Reply:
Done.